modules/dhp_connected_devices/spec/lib/fitbit_client_spec.rb

```diff
@@ -0,0 +1,64 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe DhpConnectedDevices::Fitbit::Client do
```

Comment:
I'm getting a failure on line 5 when I run this spec locally.

NameError:
  uninitialized constant DhpConnectedDevices::Fitbit
  Did you mean?  FitbitAPI
  ./modules/dhp_connected_devices/spec/lib/fitbit/fitbit_client_spec.rb:5:in `<top (required)>'

Reply:
Good catch! just pushed a fix